### ISSUE TITLE
pr (PageRank) Build error on dev-refactor

### ISSUE THREAD

**Issue by porumbes** [gunrock/gunrock#460]
**Summary:** Building with an SM_TARGET below SM60 on Maxwell architecture cards generates the following error:

```
error: no instance of overloaded function "atomicAdd" matches the argument list
            argument types are: (double *, double)
```

**Reproducibility:** Always

**Steps to Reproduce:**

1. Clone gunrock
2. Switch to the dev-refactor branch
3. Make sure to:
$ git submodule init
$ git submodule update

Set `SM_TARGETS = $(GEN_SM50)` in tests/BaseMakefile.mk
$ cd tests/pr
$ make

**Expected Results:** 

Build completes without issue.

**Actual Results:**

Build fails with the message

```
error: no instance of overloaded function "atomicAdd" matches the argument list
            argument types are: (double *, double)
```

**Configuration:**

GPU: GTX 970
CPU: Intel(R) Xeon(R) CPU E3-1280 v5 @ 3.70GHz
OS: Ubuntu 18.04.1 LTS
Cuda compilation tools, release 10.0, V10.0.130
NVIDIA-SMI 415.13       
Driver Version: 415.13       
CUDA Version: 10.0
gcc (Ubuntu 6.4.0-17ubuntu1) 6.4.0 20180424

**Comment by porumbes** [comment]
@neoblizz and I have discussed the potential fix.

**Comment by neoblizz** [comment]
Fixed by @porumbes in #461
